niworkflows/reports/figures.json

```diff
@@ -21,15 +21,15 @@
             "pattern": "[_/\\\\]acq-([a-zA-Z0-9]+)"
         },
         {
-            "name": "ce",
+            "name": "ceagent",
```

Comment:
Does this have any API effect?

Reply:
No, I do not believe so. This does not impact the recently added --bids-filter, and makes `figures.json` naming consistent with [`fmriprep.yml`](https://github.com/nipreps/niworkflows/blob/bfffaf8ea8e6fe1615be4a24089a5d750bd6e024/niworkflows/reports/fmriprep.yml#L36).

Reply:
Okay. @oesteban Is this consistent with your understanding as well?

Reply:
Yup, I also think this should not affect the API